### PR TITLE
ADAPT-2568 Add alignment boolean to Sections component

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Add alignment boolean to Sections component

# Review By (Date)
- tbd

# Review Tasks
## Setup tasks and/or behavior to test
1. Check out this branch
2. Review Storyblok config and schema for Section. Confirm that default is center alignment and enabled is left alignment.

# Associated Issues and/or People
- [ADAPT-2568](https://stanfordits.atlassian.net/browse/ADAPT-2568)

### CODE REVIEW

**Review by sherakama — APPROVED** [review]
Ok. @yvonnetangsu to review and merge when ready.

File: src/components/layout/section.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ import SbLink from "../../utilities/sbLink";
 
 const Section = ({
   blok: {
+    align,
```
Comment (yvonnetangsu) [inline src/components/layout/section.js:17]: Working great. I have one tiny nit-picky request. Could we change this boolean prop name to `isLeftAlign`? That would make it clearer in the code, and we've been naming our boolean props this way 😄  After that, GTG! 👏🏼

**Review by yvonnetangsu — CHANGES_REQUESTED** [review]
Just that one tiny request, after that GTG 👍🏼

File: src/components/layout/section.js (hunk 1 above)
Comment (rebeccahongsf) [reply src/components/layout/section.js:17]: No worries! I'm on it! 🏃‍♀️

**Review by yvonnetangsu — APPROVED** [review]
GTG - thank you 🚀

File: src/components/layout/section.js (hunk 1 above)
Comment (sherakama) [reply src/components/layout/section.js]: Good call. Thanks.

**Pull request merged**